tmail-backend/integration-tests/imap/memory-imap-integration-tests/src/test/java/com/linagora/tmail/imap/IMAPTeamMailboxIntegrationTest.java

```diff
@@ -337,4 +338,54 @@ void listShouldReturnUserMailboxWhenSUBSCRIBEDOptionAndQueryUserNamespaceInMailb
             .doesNotContain("\"#TeamMailbox.marketing.INBOX\"")
             .doesNotContain("\"Mailbox123\"");
     }
+
+    @Test
+    void memberCanCreateTopFolder() throws Exception {
```

Comment:
Just curious, can member DELETE a top mailbox?

Reply:
Tricky question, that could maybe be the object of an other ticket.

I think personally a member should not be able to dlete a mailbox that is part of the core of the team mailbox (Inbox, Draft, Trash, etc) but should be able to delete the extra folder that were created

Reply:
Agreed!

Reply:
Related to PropagateLookupRightListener issue, I created a fixup pr on upstream
https://github.com/apache/james-project/pull/2636

Reply:
> the differ can from `TMailFullDomainPathConverter`? (not include in this PR)

Hmm, I'm not sure that's the reason, but it's certainly possible

Reply:
I created a ticket for `ability to remove custom folders`
https://github.com/linagora/tmail-backend/issues/1530

Reply:
> Currently, we cannot delete folders even when we have the DeleteMailbox right, because deletion https://github.com/linagora/tmail-backend/pull/1503#discussion_r1942575339, which it does not in the case of the TeamMailboxes.

@j-hellenberg 
I fixed it in the upstream pr
https://github.com/apache/james-project/pull/2639

Reply:
Thanks!